timely_beliefs/sensors/utils.py

```diff
@@ -69,10 +69,11 @@ def unjsonify_time_dict(d: dict) -> dict:
 
 
 def func_store_list() -> dict:
-    """Returns a dictionary with function names and Callable objects supported in our function store."""
+    """Returns a dictionary with function names (incl. shorthands) and Callable objects supported in our function store."""
     functions_dict = {
         o[0]: o[1] for o in getmembers(knowledge_horizons) if isfunction(o[1])
     }
+    functions_dict = {**functions_dict, **knowledge_horizons.shorthands}
```

Comment:
Don't we now have each function twice in this dictionary? It seems this function is used only once and it would need either shorthand or full_name. Maybe shorthand could be the default way, and a parameter could change to full name if any dev needs that at any time in the future.

Also, the name of the function implies we get a list while we get a dict.

Reply:
I'll rename the function and use the shorthand as default. That would mean a breaking change, though, so a major release.

Reply:
But this function is not really being advertised as a main part of timely belief's interface. I believe it would be okay to mention in the release notes, but it's probably not necessarily major.

Reply:
I don't think the idea of assigning an ID to each knowledge horizon function is responsible for this design problem. For me it's about storing in the database anything other than the actual function name that causes the problem. All of this could be prevented by renaming the actual functions to their corresponding shorthands.